src/modules/launcher/Wox.Infrastructure/Image/ImageLoader.cs

```diff
@@ -62,6 +66,21 @@ public static void Save()
             _storage.Save(ImageCache.GetUsageAsDictionary());
         }
 
+        //Todo : Update it with icons specific to each theme.
+        public static void UpdateIconPath(Theme theme)
+        {
+            if (theme == Theme.Light || theme == Theme.HighContrastWhite)
```

Comment:
What about the other high contrast themes?

Reply:
This is currently a place holder. It would be updated once we have theme-specific icons. Currently both `if` and `else` blocks point to the same path.

Reply:
I was referring to the HighContrast1 and HighContrast2 themes. Are they all dark themed?

Reply:
Yes. High Contrast 1, High Contrast 2, and High Contrast Black are dark themed.